source/index.html

```diff
@@ -9,6 +9,9 @@
   <link rel="stylesheet" href="./styles/recipe-card.css">
   <link rel="stylesheet" href="./styles/createRecipe.css">
   <script defer src="https://use.fontawesome.com/releases/v5.15.4/js/all.js" integrity="sha384-rOA1PnstxnOBLzCLMcre8ybwbTmemjzdNlILg8O7z1lUkLXozs4DHonlDtnE7fpc" crossorigin="anonymous"></script>
+  <script src="https://kit.fontawesome.com/341af14cae.js" crossorigin="anonymous"></script>
```

Comment:
do we need this line? i thought the line above it was already importing everything from font awesome

Reply:
Oh, I tried to use only the first link. it didn't work for me, my icons are not showing. I had to make an account on font awesome and include the second line to get it working.